x-pack/plugins/endpoint/server/routes/resolver/tree.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { RequestHandler, Logger } from 'kibana/server';
+import { Fetcher } from './utils/fetch';
```

Comment:
Where is `utils/fetch` and `utils/tree`? I don't see them in this PR

Reply:
`utils/fetch` and `utils/tree` should be in the PR now.